src/directives/text-fit/text-fit.js

```diff
@@ -14,41 +14,62 @@
 //   on the shouldWarn parameter.  This allows the caller to retry again later,
 //   perhaps after the element has become visible.
 angular.module( 'gizmos.directives' ).value( 'textFit', function textFit( element, options, shouldWarn ) {
-  var min, max, mid, lastMid, containerWidth, containerHeight
-
-  element = angular.element( element )
+  var min, max, mid, lastMid, containerStyle, containerWidth, containerHeight, projectedPercentageOfBox, accuracy, allowWordWrap
+  
+  // ToDo: get options from text-fit-group
   options = options || {}
 
-  // Min and max font size.
-  min = options.min || 6
-  max = options.max || 20
+  if ( options.debug ) console.log('[textFit] Running on: ', element.text())
```

Comment:
updated so that you can use warn,info,log or whatever for the console

```
  debug = function ( ...args ) {

    if ( options.debug ) {
      
      console[ options.debug === true ? 'log' : options.debug ]( ...args )
      
    }
    
  }  

  debug( '[textFit] %sx%s in %sx%s. %s < (%s) < %s - %s', width, height, containerWidth, containerHeight, min, mid, max, isTooBig ? 'too big' : 'too small' )
 
```